CONTRIBUTING.md

```diff
@@ -105,4 +105,4 @@ Please remember read and observe the [Code of Conduct](https://github.com/cncf/f
 the [Contributor Guide](https://kubeedge.netlify.com/docs/about/contributing/) on website
 provides more detailed instructions on how to get your ideas and bug fixes seen and accepted.
 
-[kubeEdge-website-repo]: https://github.com/seattle-cloud-lab/kubeedge-website
\ No newline at end of file
+[kubeEdge-website-repo]: https://github.com/seattle-cloud-lab/kubeedge-website

```

Comment:
Change the URL to [https://github.com/kubeedge/website/](https://github.com/kubeedge/website/)

Reply:
```suggestion
[kubeEdge-website-repo]: https://github.com/kubeedge/website

```